packages/runtime-plugin-hmr/src/index.ts

```diff
@@ -6,13 +6,13 @@ import { applyHotUpdates, createHotContext } from './hot-module-state';
 import { HmrUpdatePacket, HmrUpdateResult } from './types';
 
 // TODO using host and port from the config, default to use location.host
-const port = 9801;
+const port = HMR_PORT;
 const host = 'localhost';
 
 export default <FarmRuntimePlugin>{
   name: 'farm-runtime-hmr-client-plugin',
   bootstrap(moduleSystem) {
-    console.log('[Farm HMR] connecting to the server...');
+    console.log(`[Farm HMR] connecting to the server...${port}`);
```

Comment:
Printing in a browser should not require a port number

Reply:
ok i delete it